### DISCUSSION TITLE
" Netz " dauerhaft anzeigen im Display

### DISCUSSION THREAD

**Discussion by Tomv888** [hoylabs/OpenDTU-OnBattery#1326] [Ideas]
Wurde eigentlich schonmal drüber diskutiert den Wert "Netz" dauerhaft im Display anzeigen zu lassen?Aktuell wechselt die Anzeige 
ja zwischen Netz und Gesamtertrag.

**Comment by stefan123t** [comment]
Hallo @Tomv888 schau mal hier, da habe ich eine Art Template Lösung für die Display Seiten vorgeschlagen. Da könnte man sicher auch die Anzeige-Dauer unterbringen.

https://github.com/tbnobody/OpenDTU/issues/955#issuecomment-2220989752
